src/index.html

```diff
@@ -8,6 +8,17 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css?family=Zilla+Slab:400,700" rel="stylesheet">
+
+    <!-- Google Analytics -->
+    <!-- NOTE GA tracking is disabled by default. You'll need to set the GA_TRACKING_ID build variable to enable it. -->
+    <script>
+      if (navigator.doNotTrack !== "1") {
```

Comment:
if you haven't already make sure this matches exactly with the one in hubs, or else we have to update CSP